tox.ini

```diff
@@ -49,26 +45,16 @@ whitelist_externals = {[testenv]whitelist_externals}
 commands =
     # We run the upgrade tests 1st because assuming a fast internet connection,
     # (to download the pre_built image of pulp 3.0.0), they take less long.
```

Comment:
This comment no longer applies, correct?

It referred to when we ran 1 travis job, with upgrades followed by installs.

Reply:
right!